[Reporting/Config] allow string representations of duration settings (#74202)
elastic/kibana: docs/settings/reporting-settings.asciidoc

[cols="2*<"]
|===
| `xpack.reporting.queue.pollInterval`

>   | ({ref}/common-options.html#time-units[time value]) Specifies the time that the reporting poller waits between polling the index for any

Reviewer: We don't usually put a link like this at the beginning of a sentence. Another approach is to remove "time value" and put the link on "time" in the first sentence.   In other places, you can put the link on "how long"

Reviewer: Also, use Specify instead of Specifies for consistency on the page.

  pending Reporting jobs. Can be specified as number of milliseconds. Defaults to `3s`.

| [[xpack-reporting-q-timeout]] `xpack.reporting.queue.timeout`
| ({ref}/common-options.html#time-units[time value]) How long each worker has to produce a report. If your machine is slow or under heavy

> load, you might need to increase this timeout. If a Reporting job execution time goes over this time limit, the job will be marked as a

Reviewer: ```suggestion
load, you might need to increase this timeout. If a Reporting job execution goes over this time limit, the job is marked as a
```

> failure and there will not be a download available. Can be specified as number of milliseconds. Defaults to `2m`.

Reviewer: ```suggestion
failure and no download will be available. Can be specified as number of milliseconds. Defaults to `2m`.
```

[cols="2*<"]
|===
| `xpack.reporting.queue.pollInterval`

>   | Specify the {ref}/common-options.html#time-units[time] that the reporting poller waits between polling the index for any

Reviewer 2: Might be nice to give a brief description on the order of these timeouts, so folks know what to expect with regards to the lifecycle of how these timeouts.

[cols="2*<"]
|===
| `xpack.reporting.queue.pollInterval`
  | (time value) Specifies the time that the reporting poller waits between polling the
  index for any pending Reporting jobs. Defaults to `3s`.

| [[xpack-reporting-q-timeout]] `xpack.reporting.queue.timeout`

>   | (time value) How long each worker has to produce a report. If your machine is slow or under

Author: @elastic/kibana-docs is there a good way to link `time value` to this doc: https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#time-units

[cols="2*<"]
|===
| `xpack.reporting.capture.timeouts.openUrl`
  | ({ref}/common-options.html#time-units[time value]) Specify how long to allow the Reporting browser to wait for the "Loading..." screen

>   to dismiss and find the initial data for the Kibana page. If the time is exceeded, a page screenshot is captured showing the current

Reviewer: Is "Kibana" needed before the word page?

  state, and the download link shows a warning message. Can be specified as number of milliseconds. Defaults to `1m`.

| `xpack.reporting.capture.timeouts.waitForElements`
  | ({ref}/common-options.html#time-units[time value]) Specify how long to allow the Reporting browser to wait for all visualization panels

>   to load on the Kibana page. If the time is exceeded, a page screenshot is captured showing the current state, and the download link shows

Reviewer: Remove "Kibana"?

  job, as many times as this setting. Defaults to `3`.

| `xpack.reporting.capture.loadDelay`

>   | ({ref}/common-options.html#time-units[time value]) When visualizations are not evented, this is the amount of time before taking a

Reviewer: To make this parallel with the other descriptions, you could use:

Specify the amount of time before taking a screenshot when visualizations are not evented.

[cols="2*<"]
|===
| [[xpack-reporting-csv]] `xpack.reporting.csv.maxSizeBytes`

>   | (byte size value) The maximum size of a CSV file before being truncated. This setting exists to prevent

Author: Same as https://github.com/elastic/kibana/pull/74202/files#r474158774 we should link `byte size value` to https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#byte-units
